src/services/business-lookup-services.ts

```diff
@@ -31,21 +31,22 @@ export default class BusinessLookupServices {
    * Searches for business by code or words.
    * @param query code or words to search
    * @param status status to match (ACTIVE or HISTORICAL or '' to match all statuses)
+   * @param legalTypes - the legal types we're searching for
    * @returns a promise to return the search results
    */
-  static async search (query: string, status: string): Promise<BusinessLookupResultIF[]> {
-    const legalType = 'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,CC,XL,FI,XCP,PA'
-
+  static async search (query: string, status: string, legalTypes: string): Promise<BusinessLookupResultIF[]> {
```

Comment:
This method is called by the Business Lookup shared component, which passed in the full list of legal types (unless overridden by a prop, not used in this case).

The call to search: https://github.com/bcgov/bcrs-shared-components/blob/83cfe9c8dbe19714567814eb3efd7e08f35faf49/src/components/business-lookup/BusinessLookup.vue#L237

The default prop: https://github.com/bcgov/bcrs-shared-components/blob/83cfe9c8dbe19714567814eb3efd7e08f35faf49/src/components/business-lookup/BusinessLookup.vue#L170

Where the Business Lookup component is used: https://github.com/bcgov/business-edit-ui/blob/31f470ef1f22b69e7a090aefc92b318815d139d7/src/components/common/PeopleAndRoles/OrgPerson.vue#L180 (note that `legalTypes` is not passed in, so we use the default)